validator/Makefile

```diff
@@ -41,8 +41,8 @@ OUT_IMAGE_TAG = $(OUT_IMAGE_VERSION)-$(DIST)
 OUT_IMAGE = $(OUT_IMAGE_NAME):$(OUT_IMAGE_TAG)
 
 ##### Public rules #####
-DISTRIBUTIONS := ubi8 ubuntu20.04
-DEFAULT_PUSH_TARGET := ubi8
+DISTRIBUTIONS := ubi9 ubuntu20.04
```

Comment:
Question: Do we need two flavours of validator image?

Reply:
I don't think so. I think they are all the same images

Reply:
It looks like we only ever push the tags with ubi8 suffix to the NGC prod registry

Reply:
Yeah we don't need to two flavors. Let's just stick to ubi.